### ISSUE TITLE
Is there registration method code and alignment code?

### ISSUE THREAD

**Issue by Jasolicon** [khuhm/deep-kidney-cancer#4]
Thank you for your work.
May I get registration and alignment code?
And is the classification model 2d or 3d?
Thank you.

**Comment by khuhm** [comment]
Hi, thank you for your interest in my project.
I just have uploaded the code for registration and alignment.
You can find them in the registration.py file. See function register() for registration and function crop_patch() for tumor alignment.
The classification model works in 2D.
Hope this helps.

Best,
Kwang-Hyun

**Comment by Jasolicon** [comment]
Thank you so much!
May I ask another question that how did you choose window width and window level? Did you choose different W/L for different pase?

**Comment by khuhm** [comment]
Of course, I use the range of CT pixel values in [-40, 350] HU, which covers the intensity range of kidneys and tumors in the training set. And I use the same setting for all phases.
